content/en/docs/tasks/manage-kubernetes-objects/declarative-config.md

```diff
@@ -102,7 +102,7 @@ kubectl apply -f https://k8s.io/examples/application/simple_deployment.yaml
 Print the live configuration using `kubectl get`:
 
 ```shell
-kubectl get -f https://k8s.io/examples/application/simple_deployment.yaml -o yaml
+kubectl get deploy nginx-deployment -o yaml
```

Comment:
At this point I think the original command makes sense. How about letting the reader see that both commands are equivalent?

Reply:
I agree yith you. I was in doubt whether it would be better to display the two possible commands